File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/non-plagiarized/09/t02.kt
package IR

import java.text.DecimalFormat
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    println("Enter the radius and length of a cylinder : ")
    val radius = sc.nextDouble()
    val length = sc.nextDouble()
    val area = radius * radius * 3.14159
    val volume = (radius * radius * 3.14159) * length
    val areaFormat = DecimalFormat("#.#####")
    val volumeFormat = DecimalFormat("#.#")

    println("The area is ${areaFormat.format(area)}")
    println("The volume is ${volumeFormat.format(volume)}")
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/non-plagiarized/09/t02.ts
import { Scanner } from 'readline-sync';

const sc = new Scanner();

console.log('Enter the radius and length of a cylinder : ');
const radius = sc.nextDouble();
const length = sc.nextDouble();

const area = radius * radius * 3.14159;
const volume = (radius * radius * 3.14159) * length;

const areaFormat = new Intl.NumberFormat('en-US', { minimumFractionDigits: 5, maximumFractionDigits: 5 });
const volumeFormat = new Intl.NumberFormat('en-US', { minimumFractionDigits: 1, maximumFractionDigits: 1 });

console.log(`The area is ${areaFormat.format(area)}`);
console.log(`The volume is ${volumeFormat.format(volume)}`);

